### PR TITLE
Added alpha setting to ConstSub utility

### PR DESCRIPTION
**Pull request by haandrew**
- Added ability to set alpha parameter in constituent subtractor utility 
- Protected against potential variable initialisation error in EventSubUtil

### CODE REVIEW

File: PWGJE/EMCALJetTasks/AliEmcalJetUtilityConstSubtractor.h [hunk 1]
```diff
@@ -49,6 +51,8 @@ class AliEmcalJetUtilityConstSubtractor : public AliEmcalJetUtility
   TClonesArray          *fParticlesSub;                       //!subtracted particle collection
   AliRhoParameter       *fRhoParam;                           //!event rho
   AliRhoParameter       *fRhomParam;                          //!event rhom
+  Double_t              fAlpha;                              //!pT weight exponent applied in const sub
```
Comment (aiola) [inline PWGJE/EMCALJetTasks/AliEmcalJetUtilityConstSubtractor.h]: Hello,
are you sure that you want these fields as transient members (//!) ?
This means that the content of these fields won't be kept when an instance of this class is stored in a root file (which is necessary when the task is used on the grid).
If you change them into non-transient members (i.e. remove the ! after //) remember to increase the class version (last line of this file, ClassDef).
Best,
Salvatore

**Comment by haandrew** [comment]
Hi @aiola ,

I will change these to non-transient members as this was indeed a mistake. What exactly should I increase the class version to in this case? It is currently 1..

Cheers,

Harry

**Comment by aiola** [comment]
Hello @haandrew,
it's enough to increase it to 2. The class version must be increased by one unit every time any of the non-transient member is added, deleted or changed.
Best,
Salvatore

**Comment by haandrew** [comment]
Okay thanks I have fixed the issue.

**Comment by aiola** [comment]
+1

**Pull request merged**